js/src/common/Component.ts

```diff
@@ -1,10 +1,6 @@
 import * as Mithril from 'mithril';
 
-export type ComponentAttrs = {
-  className?: string;
-
-  [key: string]: any;
-};
+export interface ComponentAttrs extends Mithril.Attributes {}
```

Comment:
What's the benefit of exporting this (and using it further below) over using `Mithril.Attributes` directly?

Reply:
We import Component in every file anyway, so that decreases the import lines we need by 1. Also, it gives us better flexibility to change things later.